miniapps/nurbs/nurbs_patch_ex1.cpp

```diff
@@ -0,0 +1,340 @@
+   int nurbs_degree_increase = 0;  // Elevate the NURBS mesh degree by this
+   int ref_levels = 0;
+   int ir_order = -1;
+
+   OptionsParser args(argc, argv);
+   args.AddOption(&mesh_file, "-m", "--mesh",
+                  "Mesh file to use.");
+   args.AddOption(&static_cond, "-sc", "--static-condensation", "-no-sc",
+                  "--no-static-condensation", "Enable static condensation.");
+   args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
+                  "--no-partial-assembly", "Enable Partial Assembly.");
+   args.AddOption(&device_config, "-d", "--device",
+                  "Device configuration string, see Device::Configure().");
+#ifdef MFEM_USE_CEED
+   args.AddOption(&algebraic_ceed, "-a", "--algebraic", "-no-a", "--no-algebraic",
+                  "Use algebraic Ceed solver");
+#endif
+   args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
+                  "--no-visualization",
+                  "Enable or disable GLVis visualization.");
+   args.AddOption(&patchAssembly, "-patcha", "--patch-assembly", "-no-patcha",
+                  "--no-patch-assembly", "Enable patch-wise assembly.");
+   args.AddOption(&reducedIntegration, "-rint", "--reduced-integration", "-fint",
+                  "--full-integration", "Enable reduced integration rules.");
+   args.AddOption(&ref_levels, "-ref", "--refine",
+                  "Number of uniform mesh refinements.");
+   args.AddOption(&ir_order, "-iro", "--integration-order",
+                  "Order of integration rule.");
+   args.AddOption(&nurbs_degree_increase, "-incdeg", "--nurbs-degree-increase",
+                  "Elevate NURBS mesh degree by this amount.");
+   args.AddOption(&compareToElementWise, "-cew", "--compare-element",
+                  "-no-compare", "-no-compare-element",
+                  "Compute element-wise solution for comparison");
+   args.Parse();
+   if (!args.Good())
+   {
+      args.PrintUsage(cout);
+      return 1;
+   }
+   args.PrintOptions(cout);
+
+   // 2. Enable hardware devices such as GPUs, and programming models such as
+   //    CUDA, OCCA, RAJA and OpenMP based on command line options.
+   Device device(device_config);
+   device.Print();
+
+   // 3. Read the mesh from the given mesh file. For this NURBS patch example,
+   //    only 3D hexahedral meshes are currently supported. The NURBS degree is
+   //    optionally increased.
+   Mesh mesh(mesh_file, 1, 1);
+   int dim = mesh.Dimension();
+
+   if (nurbs_degree_increase > 0) { mesh.DegreeElevate(nurbs_degree_increase); }
+
+   // 4. Refine the mesh to increase the resolution.
+   for (int l = 0; l < ref_levels; l++)
+   {
+      mesh.UniformRefinement();
+   }
+
+   // 5. Define a finite element space on the mesh. Here we use continuous
+   //    Lagrange finite elements of the specified order. If order < 1, we
+   //    instead use an isoparametric/isogeometric space.
+   FiniteElementCollection *fec;
+   bool delete_fec;
+   if (order > 0)
```

Comment:
since there is no option for setting the order should we remove this case?

Reply:
Removed. The positive order case is not currently supported.